.dockerignore

```diff
@@ -0,0 +1,3 @@
+# out-of-tree builds usually go here. This helps improving performance of uploading
```

Comment:
Ah nice! I always did a weird workaround to achieve that :)
Can you also add `/deps`?

Reply:
Did you add it here? Also `.git` should probably be added, shouldn't it?